I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/level_1.rpy", line 1463, in script
    label punch:#scene you punching him
ScriptError: could not find label 'idcard'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/level_1.rpy", line 1463, in script
    label punch:#scene you punching him
  File "C:\Users\49157\Downloads\renpy-8.1.3-sdk\renpy\ast.py", line 1992, in execute
    rv = renpy.game.script.lookup(target)
  File "C:\Users\49157\Downloads\renpy-8.1.3-sdk\renpy\script.py", line 1013, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'idcard'.

Windows-10-10.0.22631 AMD64
Ren'Py 8.1.3.23091805
Social Engineering Game 1.0
Tue Mar 12 13:50:25 2024
